### PR TITLE
Use Google Analytics Universal tracking code. Re: #592

### PR DESCRIPTION
**Pull request by jtsternberg**

### CODE REVIEW

File: src/@layout.latte [hunk 1]
```diff
@@ -12,15 +12,14 @@
...
-			var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-		})();
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+		ga('create', {$config->googleAnalytics}, 'auto');
+		ga('send', 'pageview');
+
```
Comment (TomasVotruba) [inline src/@layout.latte]: Could you please remove this empty line?

Comment (jtsternberg) [inline src/@layout.latte]: done.

**Comment by TomasVotruba** [comment]
Thank you @jtsternberg

**Comment by jtsternberg** [comment]
:thumbsup:

**Pull request merged**